scripts/jenkins_perf.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Run performance in jenkins build
+
+. ./env/bin/activate
+
+echo "Run performance test on $(hostname)"
+
+if [ -z %BUILD_NUMBER ]; then
+    echo "Environment variable BUILD_NUMBER is missing."
+    exit 1
+fi
+
+version=$(printf '%.8d' $BUILD_NUMBER)
+echo "Version number: $version"
+
+echo 'Before install'
+which az
+pip list
+
+build_folder=/var/build_share/$BRANCH_NAME/$version
+echo "Install build from $build_folder"
+
+python -m pip install azure-cli --find-links file://$build_folder/artifacts/build -v
+python -m pip list
```

Comment:
```
$ pip list
adal (0.4.3)
appdirs (1.4.3)
applicationinsights (0.10.0)
argcomplete (1.8.0)
asn1crypto (0.22.0)
```

```
$ pip freeze
adal==0.4.3
appdirs==1.4.3
applicationinsights==0.10.0
argcomplete==1.8.0
asn1crypto==0.22.0
```

They are both readable but advantage of `pip freeze` is you can use the output of that into `pip install -r ...`